Modules/_testcapi/file.c

```diff
@@ -4,6 +4,8 @@
 #include "parts.h"
 #include "util.h"
 #include "clinic/file.c.h"
+#include <stdio.h>
+#include <Python.h>
```

Comment:
I don't think that it's needed, parts.h includes it, no?

Reply:
yes, removed